string key
uint8 level
string name
string desc
string[] children